packages/terra-open-source-scripts/src/terra-cli/release/releaseMonoRepoHandler.js

```diff
@@ -0,0 +1,36 @@
+const childProcess = require('child_process');
+const { promisify } = require('util');
+const Logger = require('@cerner/terra-cli/lib/utils/Logger');
+
+const setupGit = require('./setupGit');
+
+const logger = new Logger({ prefix: '[terra-open-source-scripts:release]' });
+const exec = promisify(childProcess.exec);
+
+const filePath = './publish-output.txt';
+
+// Manipulate lerna publish output to find what packages were released and need to be tagged.
+const getTags = (output) => {
+  const successString = 'Successfully published:\n';
+  const lastIndexOf = output.lastIndexOf(successString);
+  if (lastIndexOf < 0) {
+    return null;
+  }
+  const substring = output.substring(lastIndexOf + successString.length);
+  const withoutDashes = substring.replace(/ - /g, '').trim();
+  return withoutDashes.split('\n');
+};
+
+module.exports = async () => {
+  const output = await exec(`npx lerna publish from-package --yes > ${filePath}`);
```

Comment:
do you still need to write this out to the filepath?

Reply:
oops forgot to remove that.

Reply:
I have removed it.